src/main/kotlin/it/unibo/protelis/web/execution/simulated/AlchemistVerticle.kt

```diff
@@ -0,0 +1,76 @@
+package it.unibo.protelis.web.execution.simulated
+
+import io.vertx.core.Context
+import io.vertx.core.Vertx
+import io.vertx.core.eventbus.EventBus
+import io.vertx.core.logging.Logger
+import io.vertx.core.logging.LoggerFactory
+import io.vertx.kotlin.coroutines.CoroutineVerticle
+import it.unibo.protelis.web.execution.CoroutineProtelisEngine
+import it.unibo.protelis.web.execution.EventBusProtelisObserver
+import java.time.LocalDateTime
+import java.time.format.DateTimeFormatter
+
+class AlchemistVerticle : CoroutineVerticle() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [UndocumentedPublicClass - [AlchemistVerticle] at /src/src/main/kotlin/it/unibo/protelis/web/execution/simulated/AlchemistVerticle.kt:14:1](https://app.codacy.com/manual/NiccoMlt/Protelis-Web/pullRequest?prid=4969516)